utils/units.js

```diff
@@ -9,3 +9,7 @@ export const formatBytes = (bytes, decimals = 2) => {
 
   return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i]
 }
+
+export const dcsToBytes = (dcs, isCellular) => {
+  return isCellular ? dcs * 20000 : dcs * 24
```

Comment:
```suggestion
  return isCellular ? dcs * 21474.83648 : dcs * 24
```

(1024^3)/(0.5/0.00001)